assets/src/components/vehiclePropertiesCard.tsx

```diff
@@ -127,12 +137,24 @@ const VehicleNearestIntersection = ({
         Current Location
       </label>
       <output className="m-current-location__value label font-s-semi" id={id}>
-        <CurrentLocation vehicle={vehicle} />
+        {isVehicle(vehicle) ? (
+          <CurrentLocation vehicle={vehicle} />
+        ) : (
+          "Exact location cannot be determined"
+        )}
```

Comment:
**question:** is handling this case actually necessary? It looks like `VehicleNearestIntersection` is only actually rendered if the `vehicleOrGhost` passes `isVehicle`, which should allow TypeScript to do type narrowing if I'm following this all correctly.

Reply:
This case is not _necessary_, but I liked it because it allowed me to use `not.toBeVisible` instead of `not.toBeInTheDocument`. I don't like `not.toBeInTheDocument` because it accepts `null`, and the functions which are compatible with that, like `queryByRole` won't help you if you typed your query wrong (e.g. it could still be in the document, you just typed the `name` wrong or some other query property) 

So by keeping it in the DOM but not visible, I'm able to test and be certain that it's not visible.

Design didn't ask for a location component which could accept a ghost, but it was easy enough that I felt it'd be okay to keep, but if it's unwanted I can change it and edit the tests and https://github.com/mbta/skate/pull/1895/files/b5e6a29dbe85586ad761c5191afd6d82bd002e91#diff-0d5230dc7a702ef8c07b6ea444bff96e31f173e04e5b36f45cff2ef60db98543R206.

Reply:
Hmm, okay. So this is an interesting tradeoff between test elegance and actual prod code elegance, in a way. I think my own decision here would probably be to not render the hidden stuff in the DOM in the ghost case, but as a reviewer I don't feel strongly about it so I trust your judgement. 👍

Reply:
I also see this as taking a component and styling it to look right, rather than generating a component to have the dom look exactly as the visual presentation looks. I'd like to/am going to keep it, because I think styling is the way to handle this, but I could be wrong!